gateway-core/components/org.wso2.carbon.gateway.core/src/test/java/org/wso2/carbon/gateway/core/exceptions/ExceptionTest.java

```diff
@@ -154,6 +161,7 @@ public void mediatorAfterTryBlockMediatorTest() throws Exception {
         resource.getDefaultWorker().addMediator(tryBlockMediator);
         resource.getDefaultWorker().addMediator(outSideCatch);
 
+        resource.setInputParamIdentifier("some_value");
```

Comment:
For a resource, we are passing a message with Identifier. This value is set in the resource object when parsing the configuration. [1] 
There for here, "some_value" is referred to the 'message' identifier. 

[1]. https://github.com/wso2/carbon-gateway-framework/pull/43/files#diff-13c7c7846b9fede8be5f29d09a00c751R420